src/extension.ts

```diff
@@ -66,10 +67,11 @@ export async function activate(context: vscode.ExtensionContext) {
 
 	// Get default commands from configuration.
 	const defaultCommands = vscode.workspace.getConfiguration("roo-cline").get<string[]>("allowedCommands") || []
+	const currentCommands = context.globalState.get<string[]>("allowedCommands")
```

Comment:
we compare the settings for the commands before force re-write. its just a check